lib/active_merchant/billing/gateways/adyen.rb

```diff
@@ -609,7 +611,7 @@ def add_reference(post, authorization, options = {})
       end
 
       def add_mpi_data_for_network_tokenization_card(post, payment, options)
-        return if options[:skip_mpi_data] == 'Y'
+        return if options[:skip_mpi_data] == 'Y' || post[:shopperInteraction] == 'ContAuth'
```

Comment:
Is this because we want to treat NetworkTokenizationCards as credit cards for subsequent transactions?